.circleci/config.yml

```diff
@@ -18,39 +18,43 @@ jobs:
       - checkout: *post_checkout
```

Comment:
If `post_checkout` is no longer used in any other job, we should move it from an anchor to just inline here.

Reply:
I left it to maintain commonality with core's usage.  If it is ever updated there it could then be easily update here as well.  Can change it if you prefer.

Reply:
I think that's a fair reason really, no need to change it IMHO.